streamlitDemo/pages/3_🔉_Text_To_Speech.py

```diff
@@ -22,7 +22,25 @@
 voice = st.radio('Select A Voice', (TTS_VOICE_BANK['voice1'], TTS_VOICE_BANK['voice2']), horizontal=True)
 
 # text input
-text = st.text_area("Enter Text", '', height = 80)
+#text = st.text_area("Enter Text", '', height = 80)
+
+# text input now a dropdown
+
+text= st.selectbox(
+'Select Text',
+(
+"I am the set of all possible positions a point can take. I may be a region in a plane or in space, but more often I am a continuous curve.Who am I?",
+"One particular element out of about the hundred or so elements in the Periodic Table is always present during my operations. I am the secret behind the relatively high boiling point of water. Who am i?",
+"Welcome to the national science and maths quiz.", "I am not one who likes much talk. I prefer action, so I'm on the side of forces. I am available when you need to turn things around and rotate objects about a point. Who am I?"
+    
+)
+)
+
+#this fxn inherently does not allow you to leave box empty
```

Comment:
what function are you referring to?

Reply:
'st.selectbox'. I made a little comments whiles working. Taken it out of the code